tests/cupyx_tests/scipy_tests/stats_tests/test_stats_py.py

```diff
@@ -0,0 +1,178 @@
+        return scp.stats.zscore(x)
+
+    @testing.numpy_cupy_allclose(scipy_name='scp', rtol=1e-3)
+    def test_zscore_1dim_float16(self, xp, scp):
+        x = testing.shaped_random((10,), xp, dtype=xp.float16)
+        return scp.stats.zscore(x)
+
+    @testing.for_all_dtypes(no_float16=True, no_bool=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
+    def test_zscore_2dim(self, xp, scp, dtype):
+        x = testing.shaped_random((5, 3), xp, dtype=dtype)
+        return scp.stats.zscore(x)
+
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-3, rtol=1e-3)
+    def test_zscore_2dim_float16(self, xp, scp):
+        x = testing.shaped_random((5, 3), xp, dtype=xp.float16)
+        return scp.stats.zscore(x)
+
+    @testing.for_all_dtypes(no_float16=True, no_bool=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-3, rtol=1e-3)
+    def test_zscore_multi_dim(self, xp, scp, dtype):
+        x = testing.shaped_random((3, 4, 5, 7), xp, dtype=dtype)
+        return scp.stats.zscore(x)
+
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-3, rtol=1e-3)
+    def test_zscore_multi_dim_float16(self, xp, scp):
+        x = testing.shaped_random((3, 4, 5, 7), xp, dtype=xp.float16)
+        return scp.stats.zscore(x)
+
+    @testing.for_all_dtypes(no_bool=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=atol)
+    def test_zscore_with_axis(self, xp, scp, dtype):
+        x = testing.shaped_random((5, 6), xp, dtype=dtype)
+        return scp.stats.zscore(x, axis=1)
+
+    @testing.for_all_dtypes(no_bool=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
+    def test_zscore_with_axis_ddof(self, xp, scp, dtype):
+        x = testing.shaped_random((2, 3, 8), xp, dtype=dtype)
+        return scp.stats.zscore(x, axis=2, ddof=2)
+
+    @testing.for_all_dtypes(no_bool=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_zscore_empty(self, xp, scp, dtype):
+        x = xp.array([], dtype=dtype)
+        return scp.stats.zscore(x)
+
+    @testing.for_dtypes('fdFD')
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_zscore_nan_policy_propagate(self, xp, scp, dtype):
+        x = xp.array([4.0, 1.0, 1.0, cupy.nan], dtype=dtype)
+        return scp.stats.zscore(x, nan_policy='propagate')
+
+    @testing.for_dtypes('fdFD')
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_zscore_nan_pocily_omit(self, xp, scp, dtype):
+        x = xp.array([4.0, 1.0, 1.0, cupy.nan], dtype=dtype)
+        return scp.stats.zscore(x, nan_policy='omit')
+
+    @testing.for_dtypes('fdFD')
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_zmap_nan_policy_omit_axis_ddof(self, xp, scp, dtype):
+        x = xp.array([4.0, 1.0, 1.0, cupy.nan], dtype=dtype)
+        return scp.stats.zscore(x, axis=0, ddof=1, nan_policy='omit')
+
+    @testing.for_dtypes('fdFD')
+    def test_zscore_nan_policy_raise(self, dtype):
```

Comment:
ditto (testing both numpy and cupy raise the same error)

Reply:
Done